documentation/source/CONTRIBUTING.md

```diff
@@ -1,88 +1,101 @@
 # Contribution Guidelines
 
-We appreciate all contributions. If you are planning to contribute back bug-fixes, please do so without any further discussion. If you plan to contribute new features, utility functions or extensions, please first open an issue and discuss the feature with us.
+Here is a simple guideline to get you started with your first contribution.
+1. Set up your environment to follow our [formatting guidelines](#code-formatting) and to use [signed-commits](#signed-commits).
```

Comment:
Add links please.

Reply:
I bet use of signed commits for external contributors could be a deal breaker. I mean if I were just a regular user and repo I'm about contribute to would require me to set up PGP agent, generate key, I'd pass. @ofrimasad is it really a requirement for _all_ contributors including non-Deci employees?

Reply:
I tend to agree, but think this is a conversation that should not bock merging this contributing.md.
Lets discuss that offline and change this part if we decide to drop it.